validate_repo.go

```diff
@@ -32,13 +32,14 @@ func ValidateHelmRepositoryURL(repoUrl string) error {
 	var client = &http.Client{
 		Timeout: time.Second * 10,
 	}
-	res, err := client.Head(url.String())
+	response, err := client.Head(url.String())
 	if err != nil {
 		return errors.Wrapf(err, invalidChartRepo, repoUrl)
 	}
 
-	// Some servers return odd responses.  We need to check more than just content length to determine failure
-	if res.ContentLength < 0 || res.StatusCode > 400 {
```

Comment:
Is the ContentLength check still needed?

Reply:
No, it seems it doesn't always get set properly so better not to rely on it.